Assets/IntegrationTests/TestShopifyCollections.cs

```diff
@@ -37,6 +38,40 @@ public class TestShopifyCollections : MonoBehaviour {
 
             Assert.IsTrue (waiter.IsStopped, Utils.MaxQueryMessage);
         }
+
+        [UnityTest]
+        public IEnumerator LoadCollectionById() {
+            ShopifyBuy.Init("351c122017d0f2a957d32ae728ad749c", "graphql.myshopify.com");
+            StoppableWaitForTime getFirstCollectionWaiter = Utils.GetWaitQuery ();
+
+            Collection foundCollection = null;
+            ShopifyBuy.Client().collections(
+                first: 1,
+                callback: (collections, error) => {
+                    getFirstCollectionWaiter.Stop();
+                    foundCollection = collections[0];
+                }
+            );
+
+            yield return getFirstCollectionWaiter;
+            Assert.IsTrue (getFirstCollectionWaiter.IsStopped, Utils.MaxQueryMessage);
+
+            StoppableWaitForTime getSecondCollectionWaiter = Utils.GetWaitQuery ();
+            var collectionIds = new List<string>() { foundCollection.id() };
```

Comment:
Can we just hardcode ids?

Reply:
Since this is also mean to work on a list of IDs, I think it would be good if the test cover 2 or more ids.